charts/brig/templates/service.yaml

```diff
@@ -10,7 +10,8 @@ metadata:
 spec:
   type: ClusterIP
   ports:
-    - port: {{ .Values.service.externalPort }}
+    - name: http
```

Comment:
Using a port name seemed safer than depending on an actual port, so I added port names to all the services.